internal/pkg/wirecli/wirek8s/wirek8s.go

```diff
@@ -68,7 +68,9 @@ func NewConfigFlags(ar util.Args) (*configflags.ConfigFlags, error) {
 	kubeConfigFlagSet := pflag.NewFlagSet("dispatcher-kube-config", pflag.ContinueOnError)
 	kubeConfigFlagSet.ParseErrorsWhitelist.UnknownFlags = true
 	kubeConfigFlagSet.SetNormalizeFunc(WordSepNormalizeFunc)
-	kubeConfigFlagSet.Set("namespace", "default")
+	if err := kubeConfigFlagSet.Set("namespace", "default"); err != nil {
```

Comment:
Above there is a note: "// IMPORTANT: If there is an error parsing flags--continue."

Should we continue to ignore the error then? I see you already ignore a similar thing in cmd/cmd.go in this PR.

Reply:
Thank you, I knew I'd done this somewhere (some tests were failing) and you've saved me the trouble of figuring out where. I've fixed it.